modules/dateUtils.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] Please do not disable linters in your project. Having linters in your project helps ensure that the quality of your written code is good.